### PR TITLE
Add Exif Orientation Plugin settings

### PR DESCRIPTION
**Pull request by arthaud-proust**
Added filepond allowImageExifOrientation properties.

https://pqina.nl/filepond/docs/api/plugins/image-exif-orientation/

### CODE REVIEW

File: packages/forms/resources/js/components/file-upload.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import * as FilePond from 'filepond'
+import FilePondPluginImageExifOrientation from 'filepond-plugin-image-exif-orientation'
```
Comment (zepfietje) [inline packages/forms/resources/js/components/file-upload.js]: Please revert these order changes.

File: packages/forms/resources/js/components/file-upload.js [hunk 2]
```diff
@@ -170,6 +172,8 @@ export default (Alpine) => {
                         },
                     })
 
+                    console.log(this.pond);
```
Comment (zepfietje) [inline packages/forms/resources/js/components/file-upload.js]: Remove this console log.

File: packages/forms/resources/views/components/file-upload.blade.php [hunk 3]
```diff
@@ -55,6 +55,7 @@
             },
             shouldAppendFiles: {{ $shouldAppendFiles() ? 'true' : 'false' }},
             shouldTransformImage: {{ $shouldTransformImage ? 'true' : 'false' }},
+            shouldOrientImageFromExif: {{ $shouldOrientImageFromExif() ? 'true' : 'false' }},
```
Comment (zepfietje) [inline packages/forms/resources/views/components/file-upload.blade.php:57]: Order alphabetically.

**Review by zepfietje — CHANGES_REQUESTED** [review]

File: packages/forms/resources/views/components/file-upload.blade.php (hunk 3 above)
Comment (arthaud-proust) [reply packages/forms/resources/views/components/file-upload.blade.php:57]: Done

**Review by zepfietje — APPROVED** [review]

File: packages/forms/src/Components/FileUpload.php [hunk 4]
```diff
@@ -237,4 +246,9 @@ public function shouldAppendFiles(): bool
     {
         return $this->evaluate($this->shouldAppendFiles);
     }
+
+    public function shouldOrientImageFromExif(): bool
+    {
+        return $this->evaluate($this->shouldOrientImageFromExif);
```
Comment (danharrin) [inline packages/forms/src/Components/FileUpload.php]:
```suggestion
        return (bool) $this->evaluate($this->shouldOrientImageFromExif);
```

**Comment by danharrin** [comment]
Thanks

File: packages/forms/src/Components/FileUpload.php (hunk 4 above)
Comment (zepfietje) [reply packages/forms/src/Components/FileUpload.php]: Nice catch.

**Pull request merged**